src/components/Card/OffersRWACard.js

```diff
@@ -0,0 +1,126 @@
+import { useNavigation } from "@react-navigation/native";
+import React, { useEffect, useState } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import { LocalSvg } from "react-native-svg";
+
+import SelectCampaignCard from "./SelectCampaignCard";
+import SelectDateCard from "./SelectDateCard";
+import { Styles } from "../../constants/Styles";
+
+const OffersRWACard = () => {
+  const [selectedCampaignOption, setSelectedCampaignOption] = useState(null);
+  const [selectedDates, setSelectedDates] = useState([null, null, null]);
+  const [offerButtonColor, setOfferButtonColor] = useState("#557184");
+
+  useEffect(() => {
```

Comment:
Remove this logic of maintaining style in useEffect.

Reply:
Updated.